tests/e2e.sh

```diff
@@ -6,34 +6,36 @@ cp ginkgolinter testdata/src/a
+[[ $(./ginkgolinter --suppress-nil-assertion=true --suppress-err-assertion=true --suppress-len-assertion=true --suppress-compare-assertion=true --suppress-async-assertion=true --forbid-focus-container=true --suppress-type-compare-assertion=true --suppress-succeed-assertion=true a/... 2>&1 | wc -l) == 235 ]]
 # suppress all but compare different types
-[[ $(./ginkgolinter --suppress-nil-assertion=true --suppress-err-assertion=true --suppress-len-assertion=true --suppress-compare-assertion=true --suppress-compare-assertion=true a/... 2>&1 | wc -l) == 269 ]]
-# allow HaveLen(0)
-[[ $(./ginkgolinter --allow-havelen-0=true a/... 2>&1 | wc -l) == 2591 ]]
+[[ $(./ginkgolinter --suppress-nil-assertion=true --suppress-err-assertion=true --suppress-len-assertion=true --suppress-compare-assertion=true --suppress-compare-assertion=true --suppress-succeed-assertion=true a/... 2>&1 | wc -l) == 285 ]]
+# allow HaveLen(0
+[[ $(./ginkgolinter --allow-havelen-0=true a/... 2>&1 | wc -l) == 2615 ]]
 # force Expect with To
-[[ $(./ginkgolinter --force-expect-to=true a/... 2>&1 | wc -l) == 2786 ]]
+[[ $(./ginkgolinter --force-expect-to=true a/... 2>&1 | wc -l) == 2810 ]]
 # suppress all - should only return the few non-suppressble
-[[ $(./ginkgolinter --suppress-nil-assertion=true --suppress-len-assertion=true --suppress-err-assertion=true --suppress-compare-assertion=true --suppress-async-assertion=true --forbid-focus-container=false --suppress-type-compare-assertion=true a/... 2>&1 | wc -l) == 155 ]]
+[[ $(./ginkgolinter --suppress-nil-assertion=true --suppress-len-assertion=true --suppress-err-assertion=true --suppress-compare-assertion=true --suppress-async-assertion=true --forbid-focus-container=false --suppress-type-compare-assertion=true --suppress-succeed-assertion=true a/... 2>&1 | wc -l) == 171 ]]
 # suppress all, force  Expect with To
-[[ $(./ginkgolinter --force-expect-to=true --suppress-nil-assertion=true --suppress-len-assertion=true --suppress-err-assertion=true --suppress-compare-assertion=true --suppress-async-assertion=true --forbid-focus-container=false --suppress-type-compare-assertion=true a/... 2>&1 | wc -l) == 778 ]]
+[[ $(./ginkgolinter --force-expect-to=true --suppress-nil-assertion=true --suppress-len-assertion=true --suppress-err-assertion=true --suppress-compare-assertion=true --suppress-async-assertion=true --forbid-focus-container=false --suppress-type-compare-assertion=true --suppress-succeed-assertion=true a/... 2>&1 | wc -l) == 794 ]]
 # enable async interval validation
-[[ $(./ginkgolinter --validate-async-intervals=true a/... 2>&1 | wc -l) == 2699 ]]
+[[ $(./ginkgolinter --validate-async-intervals=true a/... 2>&1 | wc -l) == 2723 ]]
 # suppress spec pollution
-[[ $(./ginkgolinter --forbid-spec-pollution=true a/... 2>&1 | wc -l) == 2702 ]]
+[[ $(./ginkgolinter --forbid-spec-pollution=true a/... 2>&1 | wc -l) == 2732 ]]
```

Comment:
This one goes up by 30 errors. I did not intend to add that many new errors. Maybe it's because my test data changes inadvertently had some spec pollution going on.